HW_9/index.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+class Github {
+  constructor(client, secret) {
+    this.clientId = client;
+    this.clientSecret = secret;
+  }
+
+  async getUser(userName) {
+    const data = await fetch(`https://api.github.com/users/${userName}?client_id=${this.clientId}&client_secret=${this.clientSecret}`);
+    const parsedData = await data.json();
+    return parsedData;
```

Comment:
```suggestion
    return await data.json();
```
parsedData - лишня змінна